client/package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "proxy": "http://localhost:4400",
   "dependencies": {
+    "antd": "^3.19.1",
```

Comment:
why you install this package and you don't use it?

Reply:
Used by trial time.